.github/workflows/release.yml

```
@@ -0,0 +1,454 @@
+        run: cosign sign --yes "kmgowda/sbk@$DIGEST" "ghcr.io/kmgowda/sbk@$DIGEST"
+
+  publish-packages:
+    if: ${{ !inputs.dry_run && inputs.profile == 'release' }}
+    needs: [preflight, qualify, validate-container]
+    runs-on: ubuntu-24.04
+    timeout-minutes: 120
+    permissions:
+      contents: read
+      packages: write
+    env:
+      JRELEASER_CONFIG_FILE: ${{ runner.temp }}/jreleaser-config.toml
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)